### ISSUE TITLE
Hello，I found out while running the loss is negative

### ISSUE THREAD

**Issue by mengyangz86** [HKUDS/MAERec#6]
![image](https://github.com/HKUDS/MAERec/assets/56494809/e89bf41b-f31a-464b-821a-fa820da39701)

**Comment by ai-ipy** [comment]
I also noticed this issue, and then I looked at the loss function. Cross_ The entropy function uses the logistic sigmoid function for calculation, and a negative sign is used during the calculation process, which may result in a negative loss value in the end.

<img width="286" alt="image" src="https://github.com/HKUDS/MAERec/assets/110378254/906fce79-df09-4597-b18f-29ec5e0c0e2f">

In this calculation method, if the score of the positive sample is large enough, the value of sigmoid (pos_scr) will approach 1, and the result of - t. log (t. sigmoid (pos_scr)) will be a negative number. Similarly, if the score of the negative sample is large enough, the value of sigmoid (neg_scr) will approach 0, and the result of - t. log (1- t. sigmoid (neg_scr)) will also be a negative number.

**Comment by mengyangz86** [comment]
So does that mean that it's normal for this loss to be negative

**Comment by helloelwin** [comment]
Thanks for your comments! According to our implementation (as shown in the above discussion), it is possible that the loss will be a negative value. Although it does not hurt the training, it is not aligned well with commonly used loss functions. Will fix in later versions, thanks for noticing!
